History.md

```diff
@@ -9,6 +9,8 @@ Release date: unreleased
 
 * `allow_label_click` accepts click options to be used when clicking an associated label
 * Deprecated `allow_gumbo=` in favor of `use_html5_parsing=` to enable use of Nokogiri::HTL5 when available
+* In Selenium-enabled environments, `Session#active_element` returns the element with focus (via [document.activeElement][]).
```

Comment:
This should just be for Selenium-enabled environments - there are other drivers which will end up supporting it.  Just say `Session#active_element` returns the element with focus - Not supported by the RackTest driver`